README.rst

```diff
@@ -57,52 +57,52 @@ Install dependencies to build Shmoose (example for Debian)::
 
  * sudo apt-get install zlib1g-dev libssl-dev libxml2-dev libstdc++-5-dev libqt5quick5 libqt5quickparticles5 libqt5quickwidgets5 libqt5qml5 libqt5network5 libqt5gui5 libqt5core5a qt5-default libglib2.0-dev libpthread-stubs0-dev
 
-Get Shmoose source code::
+Fetch the Shmoose source code::
 
  * cd ..
  * git clone https://github.com/geobra/harbour-shmoose
 
 Either::
 
- * open pro file within qtreator
+ * Open the .profile file within Qt Creator
```

Comment:
Not 100% sure here

Reply:
Thank you for the work.

This sentence has to be:
Open the project file (.pro file) within QtCreator

Can you please update this?